tests/integration/targets/monit/aliases

```diff
@@ -0,0 +1 @@
+destructive
```

Comment:
You need to add
```suggestion
shippable/posix/group2
destructive
```
so that it is actually run in CI. If they are not assigned to any group, they will be skipped.

Reply:
You probably want to skip some of the platforms, like add things like
```
skip/osx
skip/macos
skip/freebsd
skip/aix
```
(or a subset of that).

Reply:
ed4ce0f6